c/public/lib/source/ttread/ttread.c

```diff
@@ -357,10 +413,27 @@ struct ttrCtx_ {
         char *end;     /* Buffer end */
         char *next;    /* Next byte available */
     } src;
+    struct                  /* variable font tables */
+    {
+        /* Metrics will be computed using deltas provided for phantom points in the 'gvar' table */
+        #define VF_FLAG_HMETRICS (1 << 1)
+        long            flags;
+        float           *UDV;   /* From client */
+        Fixed           ndv[VF_MAX_AXES];               /* normalized weight vector */
+        float           scalars[VF_MAX_MASTERS];        /* scalar values for regions */
```

Comment:
As Codacy pointed out, this `scalars` attribute does seem to be unused.

Reply:
removed